client/notifications/index.jsx

```diff
@@ -28,7 +27,7 @@ import { connect } from 'react-redux';
 import analytics from 'lib/analytics';
 import config from 'config';
 import { recordTracksEvent } from 'state/analytics/actions';
-import NotificationsPanel, { refreshNotes } from 'notifications-panel';
+import NotificationsPanel, { refreshNotes } from './src/panel/Notifications';
```

Comment:
is there a plan to clean up notifications usage of casing on dir names? iirc that's a pain to clean up on a mac...